src/components/SnackChat/SnackChat.js

```diff
@@ -0,0 +1,41 @@
+import React, {Component} from 'react';
+import WebcamCapture from '../WebcamCapture/WebcamCaptureContainer';
+import PropTypes from 'prop-types';
+
+class SnackChat extends Component {
+	initialTime = new Date();
+	state = {counter: 5};
```

Comment:
if you want to use internal state in a Component you need a constructor and an initial state, it's a bit more complex than just defining state = {counter: 5}

```
constructor(props) {
    super(props);
    this.state = {counter: 5};
  }
```

inside the component class. You shouldn't modify the state directly.

https://reactjs.org/docs/state-and-lifecycle.html

Reply:
So I have set the initial state outside of the constructor before, is this a thing I shouldn't do at all or is this a special case?

Reply:
Should avoid doing, but correct it in a different PR

Reply:
(Obviously correct this instance now)